docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc

```diff
@@ -23,26 +23,31 @@ To view the status of your {fleet}-managed agents, in {kib}, go to
 [role="screenshot"]
 image::images/agent-status.png[Agents tab showing status of each {agent}]
 
-{fleet} reports these statuses:
+{agent}s can have the following statuses:
 
 |===
 
-| **Healthy** | The {agent} is online and running as expected. There are no agent
-policy updates or automatic agent binary updates in progress, but the
-agent binary may still be out of date.
+| *Healthy* | Healthy {agent}s are enrolled and checked-in. There are no agent policy updates or automatic agent binary updates in progress, but the agent binary may still be out of date. {agent}s continuously check in to the {fleet-server} 
+for required updates. 
 
-| **Unhealthy** | The {agent} is online, but requires attention because it's
-reporting either agent or integration-level errors.
+| *Unhealthy* | Unhealthy {agent}s have not checked in to {fleet-server}. At this point, you may need to address the situation. 
 
-| **Updating** | The {agent} is online and updating the agent policy, updating the
-binary, or enrolling or unenrolling from {fleet}.
+| *Updating* | Updating {agent}s are updating the agent policy, updating the binary, or enrolling or unenrolling from Fleet.
 
-| **Offline** | The {agent} has not checked in during the expected time period.
+| *Offline* | Offline {agent}s have stayed in an unhealthy status for a period of time. Offline agent's API keys remain valid. You can still see these {agent}s in the {fleet} UI and investigate them for further diagnosis if required.
 
-| **Inactive** | The {agent} is valid, but has been removed from the main {fleet} UI according to your <<set-inactivity-timeout,inactivity timeout>>.
+| *Inactive* | Inactive {agent}s have been offline for longer than the time set in your <<set-inactivity-timeout,inactivity timeout>>. These {agent}s are valid, but have been removed from the main {fleet} UI. 
```

Comment:
```suggestion
| *Inactive* | {agent}s have been offline for longer than the time set in your <<set-inactivity-timeout,inactivity timeout>>. These {agent}s are valid, but have been removed from the main {fleet} UI. 
```
For this and each of the other rows, is the first word meant to be repeated (i.e., `Inactive`, `Unenrolled`, `Updating`, `Healthy`, and `Unhealthy`? It looks to me like these terms shouldn't be repeated, but maybe I misunderstand something.

![Screenshot 2023-04-10 at 1 50 27 PM](https://user-images.githubusercontent.com/41695641/230961034-80304cb9-4a14-454d-9c56-a5563e9f81b8.png)
--
![Screenshot 2023-04-10 at 1 51 46 PM](https://user-images.githubusercontent.com/41695641/230961251-e80a7ece-94ed-4fd2-a674-d2371ea5f0ce.png)